rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -132,8 +132,10 @@ def parse_changes(lines):
             :return: dictionary of list of changes
 
             Example abipkgdiff line:
+            ================ changes of 'libp2edrvstd.so.0.0.0'===============
             '  Functions changes summary: 3 Removed, 0 Changed, 0 Added functions (4 filtered out)'
             """
+            title_re = re.compile(r'''^\s*=+\s*changes\s+of\s+\'(?P<title>.+)\'=+\s*$''', re.VERBOSE)
```

Comment:
Why multiline string on a single line? And the regexp is a bit too strict I would say. I can imagine for example that in some upcoming version of abipkgdiff the missing space after shared object filename will be added, and the regexp will stop working.

Reply:
I would use something like this:
```python
re.compile(r"^=+\s*changes\s+of\s+'(?P<filename>.+?)'.*$")
```

No need for verbosity or multiple lines with such a simple regexp.